ui-modules/blueprint-composer/app/components/catalog-selector/catalog-selector.template.html

```diff
@@ -61,7 +61,7 @@
             <!-- here and below, col-xs-3 or -4 or -2 all work giving different densities;
                  this could be configurable ("compressed"=xs-2 w no labels, "normal"=xs-3, "big"=xs-4) -->
             <div class="col-xs-3 catalog-palette-item"
-                    ng-repeat="item in searchedItems = (filterPaletteItems(items) | catalogSelectorSearch:search | catalogSelectorSort:family) | orderBy:state.currentOrder | limitTo:pagination.itemsPerPage:(pagination.page-1)*pagination.itemsPerPage track by (item.containingBundle + ':' + item.symbolicName + ':' + item.version)"
+                    ng-repeat="item in searchedItems = (filterPaletteItems(items | catalogSelectorSearch:search) | catalogSelectorSort:family) | orderBy:state.currentOrder | limitTo:pagination.itemsPerPage:(pagination.page-1)*pagination.itemsPerPage track by (item.containingBundle + ':' + item.symbolicName + ':' + item.version)"
```

Comment:
Not sure I follow how `filterPaletteItems` is being used here. Can you give a very brief description, for my benefit please?

Reply:
yes, will add as comment

`// filter by search word first, then pass to any third-party filters, so filters can adjust based on number of search resullts`